### ISSUE TITLE
NFC Card reader ACR122U possibly not working ?

### ISSUE THREAD

**Issue by lucat** [italia/cie-middleware-linux#56]
Hi there, 

trying to use the cie-middleware-linux v. 1.4.3-3 on ubuntu 20.04 with NFC card reader ACR122U does not seem to work.

Plugging in the reader results in the following messages from the kernel:

 3935.771052] usb 3-2: new full-speed USB device number 4 using xhci_hcd
[ 3935.961544] usb 3-2: New USB device found, idVendor=072f, idProduct=2200, bcdDevice= 2.14
[ 3935.961554] usb 3-2: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[ 3935.961557] usb 3-2: Product: ACR122U PICC Interface
[ 3935.961560] usb 3-2: Manufacturer: ACS
[ 3939.202546] usb 3-2: NFC: NXP PN532 firmware ver 1.6 now attached

Which seems to be fine, but then trying to use the java application CIE ID it doesn't seem to see the reader at all.

The chip on the CIE is marked C87D and the CIE was released on 26th January 2023.

**Comment by lucat** [comment]
Hi there again, 
I could solve the issue as it has nothing to do with the middleware. 
It seems the ACR122U does not work with the nfc driver. I found the hint to the solution in this page:

https://www.jamesridgway.co.uk/install-acr122u-drivers-on-linux-mint-and-kubuntu/

As the person having trouble is on ubuntu 20.04 I had to blacklist not only two but three modules. So the relevant section of his /etc/modprobe.d/blacklist.conf looks like this:

blacklist nfc
blacklist pn533
blacklist pn533_usb


At the beginning not blacklisting the last one which has dependencies caused the other two being loaded.

After that using the driver from the manufacturer for ubuntu 18.04 to be found at the link below solved the issue.

Best regards,
Luca
